omnibus/config/projects/agent.rb

```diff
@@ -21,10 +21,7 @@
   maintainer 'Datadog Packages <package@datadoghq.com>'
 end
 
-build_version do
-  source :git
-  output_format :dd_agent_format
-end
+build_version ENV['PACKAGE_VERSION']
```

Comment:
can you add a comment here to explain what sets this env var?

(ditto on the other software defs)

Reply:
Yes. At first I wanted to call the `invoke` command more explicitly but I doesn't seem possible (?). The `shellout` function is only available in the `build` blocks apparently